Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "HTTP"
 uuid = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 authors = ["Jacob Quinn", "contributors: https://github.com/JuliaWeb/HTTP.jl/graphs/contributors"]
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
Are we allowed to do a patch release when adding a new dependency? Or does that require a minor bump?

Reply:
I don't see why fixing a bug by using a dependency would be problematic.